.github/workflows/test.yml

```diff
@@ -28,6 +32,17 @@ jobs:
           forge build --sizes
         id: build
 
+      - name: Create env file
+        run: |
+          touch .env
+          echo OP_MAINNET_RPC="${{ secrets.OP_MAINNET_RPC }}" >> .env
+          echo OP_GOERLI_RPC="${{ secrets.OP_GOERLI_RPC }}" >> .env
+          echo OP_MAINNET_DEPLOYER_PK="${{ secrets.TEST_DEPLOYER_PK }}" >> .env
+          echo OP_GOERLI_DEPLOYER_PK="${{ secrets.TEST_DEPLOYER_PK }}" >> .env
+          echo ARB_GOERLI_RPC="${{ secrets.ARB_GOERLI_RPC }}" >> .env
+          echo ARB_MAINNET_RPC="${{ secrets.ARB_MAINNET_RPC }}" >> .env
+          cat .env
```

Comment:
maybe remove the cat, since we dont need to display the env

Reply:
I've removed the cat command and moved this entire "create .env" job into the env category that we had already. It runs successfully:

https://github.com/open-dollar/od-relayer/actions/runs/7213145447/job/19652349583